src/main/res/values/strings.xml

```diff
@@ -160,38 +160,38 @@
     <string name="unstable_updates">Unstable updates</string>
     <string name="unstable_updates_summary">Suggest installing unstable versions</string>
     <string name="unverified">Unverified</string>
-    <string name="update">Update</string>
-    <string name="updates">Updates</string>
-    <string name="upstream_source_code_is_not_free">The upstream source code is not free</string>
+    <string name="update">Upgrade</string>
+    <string name="updates">Upgrades</string>
+    <string name="upstream_source_code_is_not_free">The upstream source code is not libre</string>
     <string name="username">Username</string>
     <string name="username_missing">Username missing</string>
     <string name="validation_index_error_DESC">Could not validate index.</string>
     <string name="version">Version</string>
     <string name="version_FORMAT">Version %s</string>
     <string name="versions">Versions</string>
-    <string name="waiting_to_start_download">Starting download…</string>
+    <string name="waiting_to_start_download">Downloading…</string>
     <string name="whats_new">What\'s New</string>
     <string name="website">Website</string>
     <string name="prefs_language_title">Language</string>
     <string name="prefs_personalization">Personalization</string>
-    <string name="show_less">Show Less</string>
+    <string name="show_less">Show less</string>
     <string name="latest">Latest</string>
     <string name="explore">Explore</string>
-    <string name="update_all">Update all</string>
-    <string name="installed_applications">Installed applications</string>
-    <string name="sort_filter">Sort &amp; Filter</string>
-    <string name="new_applications">New applications</string>
-    <string name="install_after_sync">Install updates automatically</string>
-    <string name="install_after_sync_summary">Automatically install app updates after syncing repositories</string>
-    <string name="prefs_updated_apps">Number of updated apps to show</string>
+    <string name="update_all">Install all</string>
```

Comment:
Again, it's about updating the already installed and not installing such.

Reply:
In my head what you said means "only sync with the repos to look for new apps from a selection of already installed ones", when I think also 1.0 apps are included.
Or "Update all" means to look for new versions of all apps, which makes a lot of sense, but will get mistaken for "install" (upgrade).

Reply:
"Update all" is a button that installs the updates for the apps showing in the "updates" list (mentioned above L164)

Reply:
All the confusion is avoided, unless there is somehow something else to feasibly be installed?